### ISSUE TITLE
Get current state ready to publish on hex.pm

### ISSUE THREAD

**Issue by oubiwann** [rvirding/spell1#3]
Tasks:
* [x] Add rebar.config
* [x] Ensure the project builds with the supported versions of LFE and Erlang
* [x] Ensure Erlang spell1 works as expected/it used to
* [ ] Ensure LFE spell1 works as expected/it used to
* [ ] Update README with references to supported versions of Erlang and LFE
* [ ] Add tests
* [x] Add `make` targets for running the tests
  * [x] ensure project is compiled
  * [x] ensure testing `*.spell1` files get parsed and compiled
  * [x] ensure that when the tests run, rebar3 is able to see the `.erl` files compiled from the `*.spell1` files
* [ ] Update it to use GH Actions for CI/CD

**Comment by rvirding** [comment]
@oubiwann I have been looking at this as I was thinking of modifying the LFE syntax, only internally and it wouldn't be noticed. I then realised that I couldn't compile it as yeccscan interface had changed. I am working on that and have fixed it. While along the way I am working on the other points. I will soon release 0.3 on develop so it can actually be used and tested.

**Comment by oubiwann** [comment]
@rvirding That's sounds great!

I did make some progress here in a local branch, but had to put in on pause -- I'll take a look once you release 0.3!

**Comment by rvirding** [comment]
@oubiwann Yes, I am looking through your branch and plucking ideas from it. Btw there is now a spell1 discord server. I think I sent you an invite otherwise you can use https://discord.gg/nu5kFuXH
